libs/remixd/src/serviceList.ts

```diff
@@ -0,0 +1,3 @@
+import { RemixdClient as sharedfolder } from './services/remixdClient'
```

Comment:
May be you can use `export { RemixdClient as sharedfolder } from './services/remixdClient'`